yarn.lock

```diff
@@ -2076,9 +2076,9 @@
     "@types/react" "*"
 
 "@types/lodash@^4.14.123":
```

Comment:
Should we update the required dependency as well now? Otherwise, prop types might not get generated for consumers with stale lockfiles.